src/modules/datePickerDropdownYear.js

```diff
@@ -74,6 +74,7 @@ class DatePickerDropdownYear extends React.PureComponent {
                 <ScrollBar
                     autoHide
                     className="date-picker-dropdown-year-scrollbar"
+                    ref={ref => this.scrollComponent = ref}
```

Comment:
I think ESLINT is demanding this these days:
`ref={(ref) => { this.scrollComponent = ref; }}`